exe/kubernetes-diff

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'kubernetes-deploy/diff_task'
+require 'kubernetes-deploy/options_helper'
+require 'kubernetes-deploy/bindings_parser'
+require 'kubernetes-deploy/label_selector'
+
+require 'optparse'
+
+template_paths = []
+bindings = {}
+verbose_log_prefix = false
+selector = nil
+
+ARGV.options do |opts|
+  parser = KubernetesDeploy::BindingsParser.new
+  opts.on("--bindings=BINDINGS", "Expose additional variables to ERB templates " \
+    "(format: k1=v1,k2=v2, JSON string or file (JSON or YAML) path prefixed by '@')") { |b| parser.add(b) }
+
+  opts.on("--verbose-log-prefix", "Add [context][namespace] to the log prefix") { verbose_log_prefix = true }
+  opts.on("-f [PATHS]", Array, "comma separated list of template directories and/or filenames ( Default: current directory )") do |paths|
```

Comment:
I'm apprehensive about muddling the semantics of our default dir across different tools. To me, it would be surprising to have the same flag yield different defaults for different commands. I'm tempted to keep the default the same as deploy/render/etc. You should be able to get this for free by calling `with_validated_paths` from `options_helper.rb` (see `exe/kubernetes-deploy`, `exe/kubernetes-render`).

Reply:
This can be removed yeah, we can keep current dir default in fork, i'm glad you removed it for template_paths, was not fan of `config/deploy/env` default :)